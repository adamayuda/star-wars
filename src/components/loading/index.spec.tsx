import { Loading } from ".";
import { render } from "@testing-library/react";

describe("src/components/loading/index.tsx", () => {
  test("Render Loading component", () => {
    const renderedLoading = render(<Loading />);
    expect(renderedLoading.container).toMatchSnapshot();
  });
});
