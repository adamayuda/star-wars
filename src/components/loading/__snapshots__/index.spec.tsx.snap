// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/loading/index.tsx Render Loading component 1`] = `
<div>
  <div
    class="loading"
  >
    <img
      alt="LinkedTeam"
      src="test-file-stub"
    />
  </div>
</div>
`;
